setup.py

```diff
@@ -37,6 +37,7 @@
         "humanize>=0.5",
         "psutil>=5.5.0",
         "typing_extensions>=3.7.2",
+        "tabulate>=0.8.3",
```

Comment:
Please duplicate `tabulate==0.8.3` (strong form) in `requiremens/base.txt`.
It allows dependabot to check if everything is still going smoothly after future `tabulate` releases.